script/helperFunctions.sh

```
@@ -1514,167 +1532,284 @@ function parseTargetStateGoogleSpreadsheet() {
+      break
+    fi
+
+    # determine diamond type based on odd/even column index
+    if ((INDEX % 2 == 0)); then
+      local DIAMOND_TYPE="LiFiDiamondImmutable"
+    else
+      local DIAMOND_TYPE="LiFiDiamond"
+    fi
+
+    # get current contract version and save in variable
+    local CURRENT_VERSION=$(getCurrentContractVersion "$CONTRACT")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)